pkg/storage/export/export/export_test.go

```diff
@@ -974,8 +974,7 @@ var _ = Describe("Export controller", func() {
 			Expect(ok).To(BeTrue())
 			Expect(secret.GetName()).To(Equal(controller.getExportSecretName(testExportPod)))
 			Expect(secret.GetNamespace()).To(Equal(testNamespace))
-			secret.Name = "something"
-			return true, secret, errors.NewAlreadyExists(schema.GroupResource{}, "already exists")
+			return true, nil, errors.NewAlreadyExists(schema.GroupResource{}, secret.Name)
```

Comment:
The change looks correct to me. As in both cases we don't return error if `errors.IsAlreadyExists`, test result does not change.. I guess you found this by code inspection?